api/src/main/java/org/eclipse/microprofile/rest/client/RestClientBuilder.java

```diff
@@ -165,6 +168,43 @@ default RestClientBuilder baseUri(URI uri) {
      */
     RestClientBuilder executorService(ExecutorService executor);
 
+    /**
+     * Specifies the SSL context to use when creating secured transport connections to server endpoints
+     * from web targets created by the client instance that is using this SSL context.
+     *
+     * @param sslContext the ssl context
+     * @return the current builder with ssl context set
+     */
+    RestClientBuilder sslContext(SSLContext sslContext);
```

Comment:
Can you add `@since 1.3` to the new API methods?

Reply:
done